### PR TITLE
Support `route-type` BGP policy condition

### PR DESCRIPTION
**Pull request by wenovus**

### CODE REVIEW

**Comment by wenovus** [comment]
@DanG100 ping on this for review

**Review by DanG100 — APPROVED** [review]

File: bgp/tests/local_tests/route_type_test.go [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+					ExpectedResult: policytest.RouteAccepted,
+				}},
+				installPolicies: func(t *testing.T, dut1, dut2, dut3, dut4, dut5 *Device) {
+					installRejectPolicy(t, dut1, dut2, dut3, dut4, dut5)
+				},
+			})
+		})
+	}
+
+	test()
```
Comment (DanG100) [inline bgp/tests/local_tests/route_type_test.go:131]: why make this a var and call immediately after?

Comment (wenovus) [reply bgp/tests/local_tests/route_type_test.go:131]: Some copy-pasting artifacts. Removed.

**Comment by wenovus** [comment]
I see some flakes going on in various tests (initially aggregate, and then also bgp_propagate). I want to observe a bit more before taking the time to debug.

**Pull request merged**